### PR TITLE
Release 19.1.1

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
 
+## [19.1.1](https://github.com/theforeman/puppet-foreman/tree/19.1.1) (2022-01-25)
+
+[Full Changelog](https://github.com/theforeman/puppet-foreman/compare/19.1.0...19.1.1)
+
+**Fixed bugs:**
+
 ## [19.1.0](https://github.com/theforeman/puppet-foreman/tree/19.1.0) (2022-01-25)
 
+- Fixes [\#34317](https://projects.theforeman.org/issues/34317) - Use the correct certificate to register [\#1022](https://github.com/theforeman/puppet-foreman/pull/1022) ([ekohl](https://github.com/ekohl))
```
Comment (evgeni) [inline CHANGELOG.md:11]: this seems in the wrong section?

Comment (ekohl) [reply CHANGELOG.md:11]: I had to it manually because of my merge mistake. I did fix it in HISTORY.md not in the source :(

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [19.1.1](https://github.com/theforeman/puppet-foreman/tree/19.1.1) (2022-01-25)
```
Comment (evgeni) [inline CHANGELOG.md:3]:
```suggestion
## [19.1.1](https://github.com/theforeman/puppet-foreman/tree/19.1.1) (2022-01-26)
```

File: HISTORY.md [hunk 3]
```diff
@@ -1,3 +1,127 @@
+## [19.1.1](https://github.com/theforeman/puppet-foreman/tree/19.1.1) (2022-01-25)
```
Comment (evgeni) [inline HISTORY.md:1]:
```suggestion
## [19.1.1](https://github.com/theforeman/puppet-foreman/tree/19.1.1) (2022-01-26)
```

**Review by evgeni — APPROVED** [review]
ack pending correct date ;)

**Pull request merged**